blueocean-personalization/src/main/js/model/User.js

```diff
@@ -18,6 +19,10 @@ export class User extends Record(
     isAnonymous() {
         return false;
     }
+
+    static current() {
+        return new User(AppConfig.getInitialUser());
```

Comment:
I think `getInitialUser` is fine (actually .... I don't see the point in the "Initial" part of the name and why it's not just called `getUser`, but anyway) because getting user details is not just a security thing imo ... it's primarily an identity thing. Testing if a user has auth to do certain things would (imo) be more of a security thing.